### ISSUE TITLE
Build failure due to change in RtAudio.h location

### ISSUE THREAD

**Issue by DrChrisBoucher** [VCVRack/Rack#417]
OS (if applicable): all
Version (or "dev" if compiling from source): dev

Clean build currently fails as RtAudio.h is no longer installed into a sub-directory.

Trivial patch:

```
diff --git a/src/core/AudioInterface.cpp b/src/core/AudioInterface.cpp
index dd9b4b8..4f6b515 100644
--- a/src/core/AudioInterface.cpp
+++ b/src/core/AudioInterface.cpp
@@ -8,7 +8,7 @@
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wsuggest-override"
-#include <rtaudio/RtAudio.h>
+#include <RtAudio.h>
 #pragma GCC diagnostic pop

```

Total aside: nice catch re: the short-enums nightmare bug 8-)

Chris.

**Comment by antoniotuzzi** [comment]
great catch ! Thank you .

**Comment by AndrewBelt** [comment]
Fixed with b832b97
